setup.py

```diff
@@ -2,10 +2,21 @@
 import sys
 from datetime import datetime, MAXYEAR
 from collections import namedtuple
+import setuptools
 
-from setuptools import setup
-from setuptools.command.install import install
+try:
+    from wheel.bdist_wheel import bdist_wheel as _bdist_wheel
 
+    class bdist_wheel(_bdist_wheel):
+        def run(self):
+            raise setuptools.errors.ClassError(
+                "This is an expected error. Building wheel is disabled for the sklearn package to avoid client-side pip caching."
```

Comment:
Should we mention the brownout period and add a link to it in this error message?